src/Http/RequestMatcher/QuerystringRequestMatcher.php

```diff
@@ -45,6 +45,6 @@ public function matches(Request $request): bool
             return true;
         }
 
-        return (bool) preg_match('{'.$this->queryString.'}', rawurldecode($request->getQueryString()));
+        return (bool) preg_match('{'.$this->queryString.'}', rawurldecode($request->getQueryString() ?? ''));
```

Comment:
i think the $request must always exist and have the method, so this should be good enough:

```suggestion
        return (bool) preg_match('{'.$this->queryString.'}', rawurldecode($request->getQueryString() ?: ''));
```

Reply:
Thank, i updated my PR with your suggestion :)